.github/workflows/sep-tests.yml

```
@@ -1501,7 +1515,8 @@ jobs:
     # last also keeps a flaky -race OOM from ever obscuring the functional
     # verdict produced by the go jobs.
     needs: [ragflow_tests_infinity, ragflow_tests_elasticsearch]
-    if: ${{ github.event_name != 'pull_request' && github.event_name != 'pull_request_target' || (github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'ci') && (github.event.action != 'labeled' || github.event.label.name == 'ci')) }}
+    # Skip unless DeepDoc native backend code/build/testdata actually changed.
+    if: ${{ (github.event_name != 'pull_request' && github.event_name != 'pull_request_target' || (github.event.pull_request.draft == false && contains(github.event.pull_request.labels.*.name, 'ci') && (github.event.action != 'labeled' || github.event.label.name == 'ci'))) && needs.ragflow_preflight.outputs.has_native_deepdoc_changes == 'true' }}
```

actionlint (1.7.12)

[error] property "ragflow_preflight" is not defined in object type {ragflow_tests_elasticsearch: {outputs: {}; result: string}; ragflow_tests_infinity: {outputs: {}; result: string}}

(expression)